cmd/create/kubeletconfig/cmd.go

```diff
@@ -121,6 +117,12 @@ func run(_ *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if args.podPidsLimit > maxPidsLimit {
+		r.Reporter.Errorf("The maximum value for --pod-pids-limit is '%d'. You have supplied '%d'",
+			maxPidsLimit, args.podPidsLimit)
+		os.Exit(1)
+	}
```

Comment:
Does this block get reached? There is validation for this scenario in `GetInterativeInput`

Reply:
Yes. See my previous comment.